schemas/bashly.json

```diff
@@ -1,6 +1,17 @@
 {
     "$schema": "http://json-schema.org/draft-07/schema",
     "definitions": {
+        "custom-properties": {
+            "patternProperties": {
+                "^x-.": {
+                    "title": "custom property",
+                    "description": "A custom property",
```

Comment:
1. Can we say "A custom property or object" to hint that we support anything?
2. Does the example "hello" really add anything? I would prefer without it.

Reply:
1. I guess it's better to say `A custom property of any type`.
2. I've updated the example to `This command shows the status of all `images`, `containers` and volumes. Use with --verbose to see deleted containers.`.